src/bridge/bridge-unix-sockets.cpp

```diff
@@ -33,6 +33,7 @@
 #include <filesystem>
 
 #define TMP_DIR "/tmp"
+#define SLIMEVR_HOME_LOCAL_DIR ".local/share/slimevr"
```

Comment:
Should this use `$XDG_DATA_HOME`, which defaults to `$HOME/.local/share`

Reply:
by definition it should be `$XDG_DATA_HOME` or if it fails then `$HOME/.local/share`

Reply:
so yeah, basically you are right lmao

Reply:
discord conversation boiled down to "use the latter if the former is empty"